packs/ansible/actions/lib/ansible_base.py

```diff
@@ -21,6 +21,19 @@ def __init__(self, args):
         :type args: ``list``
         """
         self.args = args[1:]
+        self._prepend_venv_path()
+
+    @staticmethod
+    def _prepend_venv_path():
+        """
+        Modify PATH env variable by prepending virtualenv path with ansible binaries.
+        This way venv ansible has precedence over globally installed ansible.
+        """
+        venv_path = '/opt/stackstorm/virtualenvs/ansible/bin'
```

Comment:
Ah, sorry, for some reason I keep forgetting it's not a Python runner action, heh :)